### PR TITLE
Tests for Adapters need to be updated #177

### PR DESCRIPTION
**Pull request by pg301**
I have added test for NotesAdapter and have updated some previous tests for better results.
@garg-anuj @danielpassos Please review and let me know if changes are required.

### CODE REVIEW

File: mobile/src/androidTest/java/org/hawkular/client/android/adapter/AlertsAdapterTester.java [hunk 1]
```diff
@@ -60,13 +60,15 @@ public void item() {
 
         AlertsAdapter alertsAdapter = new AlertsAdapter(context, new AlertMenuAdapter(), alerts);
 
-        Assertions.assertThat(alertsAdapter).hasItem(alert, 0);
+        for(int i=0; i<alerts.size(); i++)
+        Assertions.assertThat(alertsAdapter).hasItem(alerts.get(i), i);
```
Comment (KeenWarrior) [inline mobile/src/androidTest/java/org/hawkular/client/android/adapter/AlertsAdapterTester.java]: Please stick to convention of always providing braces around blocks

**Comment by pg301** [comment]
@garg-anuj  Please review

**Comment by danielpassos** [comment]
Is this still valid?

**Comment by KeenWarrior** [comment]
@danielpassos this project is closing down due to hawkular core changes

**Pull request closed without merging**